data/food/export_agb/ingredients_base.json

```diff
@@ -58,7 +58,7 @@
         "ratio": 0.0,
         "beyondLCA": {
           "agro-diversity": 50,
-          "agro-ecology": 0.5
+          "agro-ecology": 50
```

Comment:
I'm surprised with this change, are we sure this works with the way the Elm client currently decodes this data? 🤔

Reply:
I was not sure but I blindly copied what I saw in the `plug-brightway-without-wtu` branch. 

I see that  in elm:
```
type alias Bonuses =
    { agroDiversity : Split  
    , agroEcology : Split
    , animalWelfare : Split    
    }
```
and 
```
type Split
    = Split Int
```
and `decodeBonuses` uses `decodePercent` which uses `Decode.int` and `fromPercent` and `fromPercent` expects 0 < n < 100

Reply:
Yes, I can see that, so the question is why was were these small floats before, and why was it working?